cern_access/indico_cern_access/models/access_requests.py

```diff
@@ -93,11 +99,20 @@ def has_identity_info(self):
     def has_identity_info(cls):
         return cls.birth_place.isnot(None) & cls.nationality.isnot(None) & cls.birth_date.isnot(None)
 
+    @property
+    def accompanying_persons_codes(self):
+        persons = self.registration.accompanying_persons
+        # TODO use the format_display_full_name util to generate the full name
+        persons_names = {p['id']: (f"{p['firstName']} {p['lastName']}") for p in persons}
+        return [{'name': persons_names[id], 'code': data['reservation_code']}
+                for id, data in self.accompanying_persons.items()]
+
     def clear_identity_data(self):
         self.birth_date = None
         self.nationality = None
         self.birth_place = None
         self.license_plate = None
+        self.accompanying_persons = {}
```

Comment:
should we keep the `reservation_code` and `adams_nonce` of the accompanying persons, or is it fine to clear everything?

Reply:
It's not PII so I think it makes sense to keep it - at least when sanitizing old data
when revoking a request i'd clear it